### DISCUSSION TITLE
Ignore duplicates in Tag Inputs

### DISCUSSION THREAD

**Discussion by AliMejbar** [chakra-ui/zag#78] [Q&A] [answered]
Would be good to add option to not add (ignore) duplicates in Tag Inputs : https://zagjs.com/components/react/tags-input

**Comment by anubra266** [comment, marked as answer]
@AliMejbar you can do this with the `validate` option. There's an example in the docs. https://zagjs.com/components/react/tags-input#validating-tags

**Comment by AliMejbar** [comment]
My bad , I didn't see it. Thank you !

**Reply by anubra266** [reply]
@AliMejbar Could you mark as answered, for anyone else passing by

**Reply by AliMejbar** [reply]
There isn’t the option to do it , maybe it’s because of the discussion category.

**Comment by lashdid** [comment]
I've tried using `validate={() => true}` but this didn't work, my workaround was this 

```
validate={({ value, inputValue }) => {
  setValue((prev) => [...value, inputValue]);
  return true;
}}
```

and basically makes `onValueChange` useless, it would be better if the `validate` props overrides the default duplicate checking
